backend/apps/recommendations/sync_oss_issues.py

```
@@ -0,0 +1,55 @@
+                    defaults={
+                        'title': issue['title'],
+                        'url': issue['html_url'],
+                        'repo': issue['repository']['full_name'],
+                        'labels': [label['name'] for label in issue['labels']],
+                        'is_open': True,
+                    }
+                )
+            
+            self.stdout.write(self.style.SUCCESS(f'✅ Synced {len(issues)} issues'))
+            
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)